docs/configuration/tls.md

```diff
@@ -80,19 +88,17 @@ For basic TLS, only the fields `tls.crt` and `tls.key` are needed in the kube se
             kind: Issuer
         EOF
 
-    Where `${NAMESPACE}` is the namespace where the ModelMesh Serving Service resides, and `modelmesh-serving` is the name of that service (configured via the `inferenceServiceName` global ConfigMap parameter).
-
-    Replace `modelmesh-serving-selfsigned-issuer` by the name of the issuer that you're using if needed (see previous step).
+    Above, `${NAMESPACE}` is the namespace where the ModelMesh Serving Service resides, and `modelmesh-serving` is the name of that service (configured via the `inferenceServiceName` [global ConfigMap](./README.md). parameter). You can also replace `issuerRef.name` to match the name of the issuer used above if necessary.
 
-    `${HOSTNAME}` is optional but should be set as follows when configuring an external Kubernetes Ingress or OpenShift route as described [here](./README.md#exposing-an-external-endpoint-using-an-openshift-route):
+    `${HOSTNAME}` is optional, but should be set when configuring an external Kubernetes Ingress or OpenShift route as described [here](./README.md#exposing-an-external-endpoint-using-an-openshift-route):
 
-        HOSTNAME=`oc get route modelmesh-serving -o jsonpath='{.spec.host}'`
+           HOSTNAME=`oc get route modelmesh-serving -o jsonpath='{.spec.host}'`
 
-    If the certificate request is successful, a TLS secret with the PEM-encoded certs will be created as `${SECRET_NAME}`.
+    If the certificate request is successful, a TLS secret with the PEM-encoded certs will be created as `modelmesh-serving-cert`, unless changed above.
 
 4.  Wait for the certificate to be successfully issued
 
-        kubectl get certificate/${SECRET_NAME} --watch
+        kubectl get certificate/modelmesh-serving-cert --watch
 
     Once you see `Ready` as `True`, proceed to the next step.
 
```

Comment:
the output below should show the value of `${SECRET_NAME}`  as we define it earlier in this doc